### PR TITLE
Add a listener functional test for #6256

### PR DESCRIPTION
**Pull request by johndodev**
As requested here https://github.com/doctrine/doctrine2/issues/6256#issuecomment-275140592 there is a failing test about not executed listener, which seems is a bug.

### CODE REVIEW

**Comment by lcobucci** [comment]
Related tests #1085 (for different listener though)

File: tests/Doctrine/Tests/ORM/Functional/Ticket/DDC6256Test.php [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+{
+    /**
+     * @Id
+     * @Column(type="integer")
+     * @GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * OneToOne(targetEntity="StoredFile", orphanRemoval=true, cascade={"all"})
```
Comment (malukenho) [inline tests/Doctrine/Tests/ORM/Functional/Ticket/DDC6256Test.php]: Missing lead `@`

**Comment by johndodev** [comment]
Haha and with the proper annotation the test is passing.. I guess the PR can be closed.

**Comment by Ocramius** [comment]
Closing as `invalid`/`incomplete`

File: tests/Doctrine/Tests/ORM/Functional/Ticket/DDC6256Test.php (hunk 1 above)
Comment (Ocramius) [inline tests/Doctrine/Tests/ORM/Functional/Ticket/DDC6256Test.php]: Hawk eye!

**Comment by lcobucci** [comment]
@johndodev closing then, thanks for the effort :wink:

**Pull request closed without merging**